docker/mongodb-kubernetes-tests/tests/multicluster/multi_cluster_scale_up_cluster.py

```diff
@@ -17,6 +25,26 @@
 BUNDLE_SECRET_NAME = f"prefix-{RESOURCE_NAME}-cert"
 
 
+@pytest.fixture(scope="module")
+def project_name_prefix(namespace: str) -> str:
+    return random_k8s_name(f"{namespace}-project-")
+
+
+@pytest.fixture(scope="module")
+def new_project_configmap(namespace: str, project_name_prefix: str) -> str:
+    cm = read_configmap(namespace=namespace, name="my-project")
+    project_name = f"{project_name_prefix}-new-project"
+    return create_or_update_configmap(
```

Comment:
again, reconfiguring the project! We need to followup on this [PR](https://github.com/mongodb/mongodb-kubernetes/pull/173)... 
I'm leaving the comment there to adjust it in here as well.